.ci/ci.yml.tmpl

```diff
@@ -268,29 +268,6 @@ jobs:
                 get_params:
                   skip_clone: true
 
-    - name: ansible-test
```

Comment:
This is a temporary measure until everything works nicely on with their test runner. I'll file an issue to reintroduce unit testing back into the Magician